tools/topology/topology2/platform/intel/dmic-wov.conf

```diff
@@ -0,0 +1,207 @@
+Object.Dai {
```

Comment:
I'd squash this patch with the previous 2 so that everyone know what you are doing